### PR TITLE
Improve UUFClient side fragment rendering logic to include JS and CSS…

### LINKED ISSUE #37: Improve UUFClient related fragment rendering logic to include JS and CSS resources
Related UUF Issue: [wso2/carbon-uuf/#129](https://github.com/wso2/carbon-uuf/issues/129)

### PR DESCRIPTION
**Pull request by cnapagoda**
Improve UUFClient side fragment rendering logic to include JS and CSS resources

RESOLVE #37

### CODE REVIEW

File: samples/apps/org.wso2.carbon.uuf.common.sample-app.app/src/main/fragments/click/click.hbs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{{js "js/click.js"}}
+{{css "css/click.css"}}
+<div id="{{#if @params.id}}{{@params.id}}{{else}}message1{{/if}}"
+                                                        class="message message-{{@params.msgClass}} {{@params.class}}">
```
Comment (cnapagoda) [inline samples/apps/org.wso2.carbon.uuf.common.sample-app.app/src/main/fragments/click/click.hbs]: Don't we need to have 120 line space for HBS file?

Comment (this) [inline samples/apps/org.wso2.carbon.uuf.common.sample-app.app/src/main/fragments/click/click.hbs]: yep, my complain is `class="message ...` part is way too indented

File: components/org.wso2.carbon.uuf.common.foundation.ui/src/main/fragments/uuf-client/public/uuf-client/uuf-client.js [hunk 2]
```diff
@@ -34,9 +34,14 @@ var UUFClient = {};
...
      * @type {{string: {start: Object, end: Object}}}
      */
-
     var zonesMap;
 
+    /**
+     * Array of rendered fragment names.
+     * @type {Array} Array of fragment fully qualified names
+     */
+    var renderedFargments = [];
```
Comment (cnapagoda) [inline components/org.wso2.carbon.uuf.common.foundation.ui/src/main/fragments/uuf-client/public/uuf-client/uuf-client.js]: spelling error

Comment (this) [inline components/org.wso2.carbon.uuf.common.foundation.ui/src/main/fragments/uuf-client/public/uuf-client/uuf-client.js]: good catch

**Review by this — APPROVED** [review]

**Pull request merged**